### PR TITLE
♻() refactor token contract and interface

### PR DESCRIPTION
**Pull request by Joachim-Lebrun**

### CODE REVIEW

File: contracts/token/IToken.sol [hunk 1]
```diff
@@ -1,7 +1,11 @@
 pragma solidity ^0.6.0;
 
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "../registry/IIdentityRegistry.sol";
+import "../compliance/ICompliance.sol";
+
 //interface
-interface IToken {
+interface IToken is IERC20{
```
Comment (Nakasar) [inline contracts/token/IToken.sol]:
```suggestion
interface IToken is IERC20 {
```

File: contracts/token/IToken.sol [hunk 2]
```diff
@@ -10,7 +14,24 @@ interface IToken {
     function onchainID() external view returns (address);
     function symbol() external view returns (string memory);
     function version() external view returns (string memory);
+    function getIdentityRegistry() external view returns (IIdentityRegistry);
```
Comment (Nakasar) [inline contracts/token/IToken.sol]: Better name I think:

```suggestion
    function identityRegistry() external view returns (IIdentityRegistry);
```

File: contracts/token/IToken.sol [hunk 3]
```diff
@@ -10,7 +14,24 @@ interface IToken {
...
+    function getCompliance() external view returns (ICompliance);
```
Comment (Nakasar) [inline contracts/token/IToken.sol]: Better name for a getter I think:
```suggestion
    function compliance() external view returns (ICompliance);
```

File: package.json [hunk 4]
```diff
@@ -15,12 +15,12 @@
   ],
   "scripts": {
     "build": "truffle compile && truffle-flattener contracts/token/Token.sol > build/TokenFlat.sol && truffle-flattener contracts/registry/IdentityRegistry.sol > build/IdentityRegistryFlat.sol && truffle-flattener contracts/registry/ClaimTopicsRegistry.sol > build/ClaimTopicsRegistryFlat.sol && truffle-flattener contracts/registry/TrustedIssuersRegistry.sol > build/TrustedIssuersRegistryFlat.sol",
-    "coverage": "solidity-coverage",
-    "test": "solidity-coverage",
+    "coverage": "truffle run coverage",
+    "test": "(ganache-cli 2>&1 > /dev/null &) && truffle test && kill $(ps -ax | grep \"ganache-cli\" | grep -v grep | cut -d' ' -f2)",
```
Comment (Myna65) [inline package.json]: Why do you launch `ganache-cli` manually. `truffle test` do it automatically :thinking: 
```suggestion
    "test": "truffle test",
```

**Pull request merged**